test/cpp/qps/run_perf_db_test.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python
```

Comment:
I believe that this file does not belong in the grpc repository at all. This should be in grpc-tools, as it introduces far too many dependences (IMO) to be used well in grpc. You need to have netperf, etc., for this to be sensible. Truth be told, I would just as well see all the support infrastructure (PerfDBReporter class, etc) moved out of grpc and into grpc-tools. but that is less critical as it's still somewhat consistent with the overall repository.